### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by aeksco** [helloresolven/rested#66]
No updates since 2013 - is this going to see any new features or maintenance going forward?

If RESTed is abandoned, would it be possible to open-source the project so others may continue to develop new features?

I'm currently using the tool to debug a PDF generator on my server and I would like to be able to set my default `Display Option` to `Rendered HTML`. It's _very_ annoying to have to hit that dropdown each time I make a request.

**Screenshot for clarity:**
<img width="969" alt="display_options" src="https://cloud.githubusercontent.com/assets/4616233/13710260/65517948-e786-11e5-8dbf-310c6815d61b.png">

**Comment by Tkizzy** [comment]
Yes, I love rested, but there's so much that would make it awesome. For example, an iPad version with synced requests between the two!

**Comment by gaetandezeiraud** [comment]
open-source is a good solution!
